templates/mail/alarm.vm

```diff
@@ -4,7 +4,7 @@
 <body>
 Device: $device.name<br>
 Alarm:  $position.getString("alarm")<br>
-Time: $event.serverTime<br>
+Time: $dateTool.format("YYYY.MM.dd HH:mm:ss", $event.serverTime, $locale, $timezone)<br>
```

Comment:
I think hyphens instead of dot would be more universal in this case. Applies for all templates.